libr/core/cmd.c

```diff
@@ -1821,6 +1822,18 @@ static int cmd_resize(void *data, const char *input) {
 			}
 		}
 		return true;
+	case 'j': { // "rj"
+			PJ * pj = pj_new ();
+			pj_o (pj);
+			if (oldsize != -1) {
+				pj_kn (pj, "size", oldsize);
+			}
+			pj_end (pj);
+			char *s = pj_drain (pj);
+			r_cons_printf ("%s\n", s);
```

Comment:
Use r_cons_println instead, no need for formatting.

Reply:
Replaced.